tests/integration/tx/aminojson/aminojson_test.go

```diff
@@ -459,15 +458,15 @@ func postFixPulsarMessage(msg proto.Message) {
 // using encoding/json.  This hacky way of sorting JSON fields was used by the legacy amino JSON encoding in
 // x/auth/migrations/legacytx.StdSignBytes.  It is used here ensure the x/tx JSON encoding is equivalent to
 // the legacy amino JSON encoding.
-func sortJson(bz []byte) ([]byte, error) {
-	var c any
-	err := json.Unmarshal(bz, &c)
-	if err != nil {
-		return nil, err
-	}
-	js, err := json.Marshal(c)
-	if err != nil {
-		return nil, err
-	}
-	return js, nil
-}
+// func sortJson(bz []byte) ([]byte, error) {
```

Comment:
instead of commenting, could you delete?

Reply:
OK， I will contribute more valuable functions codes.